uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -14,26 +14,26 @@ class CRM_HRCore_Upgrader extends CRM_HRCore_Upgrader_Base {
    * @var array
    *   List of jobs in CiviCRM and their intended default status in the form [job_name => is_active]
```

Comment:
Since the array format has changed, you have to update the docblock as well. Since now the variable names are more self descriptive, I believe only the `@var array` part is enough